reason-react-native/src/apis/PanResponder.md

```diff
@@ -1,9 +1,137 @@
 ---
-id: apis/Packager
-title: Packager
-wip: true
+id: apis/PanResponder
+title: PanResponder
 ---
 
+`PanResponder` allows more fine-grained control of the _Gesture Responder
+System_, as the handlers can depend also on `gestureState`, in addition to the
+`Event.pressEvent` which are passed to the _Gesture Responder System_ handlers.
+In essence, by means of a custom `PanResponder`, _Gesture Responder System_
+handlers are extended to also depend upon the `gestureState`.
+
+## Methods
+
+- `create` allows creation of a `PanResponder` given a `config`
```

Comment:
Instead of single list item (because that's how it's rendering the way it is), I think we should go with normal title ``### `method` ``. This will allows anchor links & will be better with incoming search engine :)